src/main/java/io/camunda/testing/assertions/JobAssert.java

```diff
@@ -0,0 +1,105 @@
+package io.camunda.testing.assertions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.api.response.ActivatedJob;
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.MapAssert;
+import org.assertj.core.data.Offset;
+
+// TODO discuss name: job assertions, service task assertions, worker assertions
+
+/** Assertions for {@code ActivatedJob} instances */
+public class JobAssert extends AbstractAssert<JobAssert, ActivatedJob> {
```

Comment:
🔧 To stay in line with the other assertions this class should be renamed to `JobAssertions`.
💭 AssertJ seems to go with the XAssert style. Maybe we should renamed the other classes to stay in line with AssertJ.

Reply:
see above